docs/source/contributing/writing-docs.adoc

```diff
@@ -104,7 +103,13 @@ $ make serve_docs
 ----
 
 The `make serve_docs` command builds and generates a live-preview of the documentation.
-You can access the documentation by browsing to *http://<IP-of-Docker-daemon>:4567*.
+You can access the documentation by browsing to *\http://<IP-of-Docker-daemon>:4567*.
+
+To make verify all links (external as well as internal ones), run:
```

Comment:
@hferentschik , I think you forgot to remove `make` here.

I suggested, `To verify all links ...` :smile: